klpbuild/ksrc.py

```diff
@@ -444,17 +444,12 @@ def get_patched_kernels(self, codestreams, commits, cve):
                 if not patched and kernel not in suse_tags:
                     continue
 
-                print(f"\n{cs.name()} ({kernel}):")
-
                 # If no patches/commits were found for this kernel, fallback to
                 # the commits in the main SLE branch. In either case, we can
                 # assume that this kernel is already patched.
-                for c in kern_commits if patched else suse_commits:
-                    print(f"{c}")
```

Comment:
Do we really need to remove this prints? It helps to track the logic taken by klp-build.